packages/block-editor/src/components/block-list/content.scss

```diff
@@ -454,3 +454,15 @@ _::-webkit-full-page-media, _:future, :root .has-multi-selection .block-editor-b
 		margin-bottom: auto;
 	}
 }
+
+.block-editor-block-list__zoom-out-separator {
+	/* same color as the iframe's background */
+	background: $gray-300;
```

Comment:
This probably isn't a huge issue, but I noticed a _very_ subtle sub pixel line of the site's background image poking through while testing:

<img width="629" alt="image" src="https://github.com/user-attachments/assets/5526d7f1-5e7a-4775-9a91-a8eef4a9e156">

Here it is zoomed in, in case it helps:

<img width="879" alt="image" src="https://github.com/user-attachments/assets/ee58b59a-968e-4c68-89a9-803d65f5da4a">

I imagine this might be because we're scaled down to 50% so the background color here doesn't perfectly extend over the background image?

Reply:
Good spot. We'll have to look into that one.

Reply:
✅ I fixed this by adding an additional 1px

Reply:
Nice idea 👍